assets/src/application/ui/input/SwitchInput/SwitchInput.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { SwitchProps } from 'antd/lib/switch';
+import { Switch } from 'antd';
+
+import { BaseInput } from '../BaseInput';
+
+interface SwitchInputProps extends SwitchProps {
+	label: string;
+}
+
+const SwitchInput: React.FC<SwitchInputProps> = React.memo(({ label, className, ...rest }) => {
+	return (
+		<BaseInput label={label} className={className}>
+			<Switch {...rest} />
```

Comment:
this have a premium feel/ touch, nice one:

![image](https://user-images.githubusercontent.com/1477453/77060338-150db780-69e1-11ea-9295-d3829c148b65.png)